test/functional/page_objects/visualize_page.js

```diff
@@ -364,7 +364,10 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     async setInspectorTablePageSize(size) {
       const panel = await testSubjects.find('inspectorPanel');
       await find.clickByButtonText('Rows per page: 20', panel);
-      await find.clickByButtonText(`${size} rows`, panel);
+      // Table size buttons are in popover element that is not under the inspectorPanel
+      // but are in an it's own portal attached directly to the body
```

Comment:
Can we rephrase this? I didn't know what "Table size buttons" meant until I figured out the context -- at first I thought it was a reference to buttons that are table-sized.

```js
// The buttons for setting table page size are in a popover element. This popover
// element appears as if it's part of the inspectorPanel but it's really attached
// to the body element by a portal.
```